backend/app.py

```diff
@@ -172,6 +177,47 @@ class CsetsUpdate(BaseModel):
     row_index_data_map: Dict[int, Dict[str, Any]] = {}
 
 
+@app.get("/concept-set-names")
+@app.get("/datasets/csets/names")
+@app.get("/jq-cset-names")
+def cset_names() -> Union[Dict, List]:
```

Comment:
@Sigfried I added 3 different names for this route. I checked, and all of them work:

```py
@app.get("/concept-set-names")
@app.get("/datasets/csets/names")
@app.get("/jq-cset-names")
```

These are all actually aliases. They do a call to: `csets_read(field_filter=['conceptSetNameOMOP'])`, which does the work.

If we want to keep these aliases (which I think is nice, honestly), my preference is for `concept-set-names`, but there's not much harm in keeping these other ones. It will bloat what's returned from `/`, though. We probably don't want the user to see all these aliases if we only intend for them to use one of them. But we can always filter these out manually from the response for `/` if we want.